gtk2_ardour/add_route_dialog.cc

```diff
@@ -598,11 +591,31 @@ AddRouteDialog::type_wanted()
 	} else if (str == _("Foldback Busses")) {
 		return FoldbackBus;
 	} else {
-		assert (0);
 		return AudioTrack;
 	}
 }
 
+std::string
+AddRouteDialog::type_wanted_to_string(AddRouteDialog::TypeWanted type_wanted) {
+	switch(type_wanted) {
+		case AudioBus:
+			return _("Audio Busses");
+		case MidiBus:
+			return _("MIDI Busses");
+		case MidiTrack:
+			return _("MIDI Tracks");
+		case AudioTrack:
+			return _("Audio Tracks");
+		case VCAMaster:
+			return _("VCA Masters");
+		case FoldbackBus:
+			return _("Foldback Busses");
+		default:
+			return _("Audio Tracks");
+			break;
+	}
+}
```

Comment:
These should probably not be translatable. In general the state should not contain translatable strings. We use `X_("..")` to indicate that.